doc/install.rst

```diff
@@ -58,61 +86,96 @@ PyGMT requires the following libraries:
+Before installing PyGMT, we must install GMT itself along with the other
+dependencies. The easiest way to do this is via the ``conda`` package manager.
 We recommend working in an isolated
 `conda environment <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`__
-to avoid issues with competing versions of its dependencies.
+to avoid issues with conflicting versions of dependencies.
 
 First, we must configure conda to get packages from the
 `conda-forge channel <https://conda-forge.org/>`__::
 
     conda config --prepend channels conda-forge
 
-Now we can create a new conda environment with Python and all our dependencies installed
-(we'll call it ``pygmt`` but you can change it to whatever you want)::
+Now we can create a new conda environment with Python and all our dependencies
+installed (we'll call it ``pygmt`` but feel free to change it to whatever you
+want)::
 
      conda create --name pygmt python=3.9 pip numpy pandas xarray netcdf4 packaging gmt
 
-Activate the environment by running::
+Activate the environment by running the following (**do not forget this step!**))::
 
     conda activate pygmt
 
-From now on, all commands will take place inside the conda virtual environment and won't
-affect your default installation.
+From now on, all commands will take place inside the conda virtual environment
+called 'pygmt' and won't affect your default 'base' installation.
 
 
 Installing PyGMT
 ----------------
 
-Now that you have GMT installed and your conda environment activated, you can
-use ``conda`` to install the latest release of PyGMT from `conda-forge <https://anaconda.org/conda-forge/pygmt>`__::
+Now that you have GMT installed and your conda virtual environment activated,
+you can install PyGMT using any of the following methods:
+
+Using conda (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This installs the latest stable release of PyGMT from
+`conda-forge <https://anaconda.org/conda-forge/pygmt>`__::
 
     conda install pygmt
 
-or use ``pip`` to install from `PyPI <https://pypi.org/project/pygmt>`__::
+Using pip
+~~~~~~~~~
+
+This installs the latest stable release from
+`PyPI <https://pypi.org/project/pygmt>`__::
 
     pip install pygmt
 
-Alternatively, you can install the development version from the GitHub repository::
+Alternatively, you can install the latest development version from
+`TestPyPI <https://test.pypi.org/project/pygmt>`__::
+
+    pip install --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pygmt
+
+or from PyGMT's GitHub repository (slower as it downloads the whole archive)::
 
-    pip install https://github.com/GenericMappingTools/pygmt/archive/master.zip
+    pip install git+https://github.com/GenericMappingTools/pygmt.git#egg=pygmt
 
-This will allow you to use the ``pygmt`` library from Python.
+Any of the above methods (conda/pip) should allow you to use the ``pygmt``
```

Comment:
What about cloning the PyGMT repository? I feel that it may improve the visibility of PyGMT source codes, thus attract more potential contributors.
```
git clone https://github.com/GenericMappingTools/pygmt
cd pygmt
make install
```

Reply:
You sure we want to put this on the same page? I feel like the installation instructions already look way too complicated.

Reply:
OK, let's leave it out.